spec/spec_helper_acceptance_local.rb

```diff
@@ -0,0 +1,59 @@
+# rubocop:disable Style/AccessorMethodName
+
+require 'serverspec'
+require 'puppet_litmus'
+require './spec/support/acceptance/helpers.rb'
```

Comment:
I don't think you need the `'./'` part of the path. Paths under `spec` are automatically added to `$LOAD_PATH` when running rspec.

Reply:
hmm, I'm getting a load error when I remove the `./`.

Reply:
`support/acceptance/helpers.rb` seems to be working.